app/src/main/java/com/android_animation_arsenal_example/MainActivity.java

```diff
@@ -1,35 +1,233 @@
 package com.android_animation_arsenal_example;
 
+import android.app.ActivityOptions;
+import android.content.Intent;
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
-import android.support.design.widget.Snackbar;
+import android.support.v4.app.ActivityCompat;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
 import android.view.View;
 import android.view.Menu;
 import android.view.MenuItem;
+import android.view.Window;
+import android.view.animation.Animation;
+import android.widget.Button;
+import android.widget.ImageView;
+
+import com.android_animation_arsenal.AnimationArsenal;
+import com.android_animation_arsenal.views.RevealDialog;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
 public class MainActivity extends AppCompatActivity
 {
+    @Bind(R.id.circularReveal_iv) ImageView mImageViewReveal;
+    @Bind(R.id.specificResource_iv) ImageView mImageViewSpecified;
+    @Bind(R.id.fadeInResource_iv) ImageView mImageViewFadeIn;
+    @Bind(R.id.fadeOutResource_iv) ImageView mImageViewFadeOut;
+    @Bind(R.id.scaleResource_iv) ImageView mImageViewScale;
+    @Bind(R.id.slideLeftResource_iv) ImageView mImageViewSlideLeft;
+    @Bind(R.id.slideRightResource_iv) ImageView mImageViewSlideRight;
```

Comment:
:eyes: `_iv`-- @roideuniverse @asolis90 I remember we had a discussion about this postfixing of the IDs before... What was the output of that conversation again?